json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -207,6 +210,12 @@ public function update_settings() {
 						$updated[ $key ] = $value;
 					};
 					break;
+				case 'jetpack_protect_whitelist':
+					$result = jetpack_protect_save_whitelist( $value, false );
+					if ( ! $result ) {
```

Comment:
Should we also check for `is_wp_error()` for the result, or will it always just evaluate to false if we're rejecting?